source/kmip.rst

```diff
@@ -89,5 +89,25 @@ Alternatively, you can start |PSMDB| using the command line as follows:
      --kmipClientKeyFile <path_to_client_private_key> \
      --kmipKeyIdentifier <kmip_identifier>
 
+.. _upgrade-kmip:
+
+Minor upgrade of |PSMDB| to version 5.0.11-10 and higher
+========================================================
+
+While the data-at-rest encryption using the KMIP Protocol is in the tech preview stage, we recommend using it only for technical purposes as breaking changes can be introduced. With the ``kmipKeyIdentifier`` option becoming optional in version 5.0.11, the standard upgrade procedure doesn’t work. 
+
+If you do need to upgrade |PSMDB| to version 5.0.11-10 and higher, this section provides the upgrade steps.
+
+For a single-node deployment, use the ``mongodump`` / ``mongorestore`` tools. 
+
+For replica sets, go through the `encrypting existing data steps <https://www.mongodb.com/docs/v5.0/tutorial/configure-encryption/#std-label-encrypt-existing-data>`_ during the upgrade as follows: 
```

Comment:
For a single-node deployment, use the ``mongodump`` / ``mongorestore`` tools to make backup before update and restore it after binaries are updated.

For replica sets, it will be required to re-encrypt data with NEW key during upgrade. Go through the `encrypting existing data steps <https://www.mongodb.com/docs/v5.0/tutorial/configure-encryption/#std-label-encrypt-existing-data>`_  as follows: